### ISSUE TITLE
Cant drag to empty group

### ISSUE THREAD

**Issue by Beregoz** [billvenhaus/ReorderableCollectionView.Forms#3]
Pretty much the title. If the group is empty, an item can not be dragged to it from another group, even if "CanMixGroups" is set to true. It should be possible to drop the item to the footer or header of group.

**Comment by billvenhaus** [comment]
Yeah, this is an issue. Unfortunately I don't know if there is an easy/clean fix. The proposed solution in this discussion I found on Stack Overflow is to add a temporary dummy item when the drag process starts. Not sure if I'm too wild about that but I can investigate.

https://stackoverflow.com/q/35920874
https://developer.apple.com/forums/thread/114010
